addOns/exim/src/main/resources/org/zaproxy/addon/exim/resources/Messages.properties

```diff
@@ -44,11 +44,17 @@ exim.importLogFiles.choosefile.title = Select Input Type
 exim.importLogFiles.import.menu.label = Import URLs from Logs or raw Files...
 exim.importLogFiles.log.type.modsec2 = ModSecurity2 Logs
 exim.importLogFiles.log.type.zap = ZAP Messages
+exim.importPcap/Pcapng.choosefile.filter.pcap.description = Pcap file (*.pcap)
+exim.importPcap/Pcapng.choosefile.filter.pcapng.description = Pcap file (*.pcapng)
+
+exim.importPcap/Pcapng.topmenu.import = Import a PCAP or PCAPNG file
+exim.importPcap/Pcapng.topmenu.import.tooltip = Import a PCAP or PCAPNG file and add the HTTP(S) traffic inside to the sites tree and history panel.
```

Comment:
For thought/discussion do we really need to say PCAP or PCAPNG I know they're technically different but does the greater "industry" treat them as one in the same so to speak?

Reply:
Great question, I personally have no clue if people will deduce that if only "Import a PCAP file" is shown. But I agree it's not great to have such a long title. I thought maybe using "Import a PCAP(NG) file" could help but not sure it is an elegant solution either